sdk/appconfiguration/Azure.Data.AppConfiguration/samples/Sample3_LockUnlockSetting.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+
+using System;
+using NUnit.Framework;
+
+namespace Azure.Data.AppConfiguration.Samples
+{
+    [Category("Live")]
```

Comment:
I think we have a new attribute for live tests.

Reply:
Only for tests that already take advantage of the recording infrastructure, though. IIRC, appconfig samples didn't use it.